lib/scrabble/player.rb

```diff
@@ -0,0 +1,122 @@
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :tiles
+
+
+    ## -------------------------------------------------------------------------
+    # CLASS METHODS
+
+
+    def initialize(name)
```

Comment:
That's really good to know! I was confused after reading the requirements for the project, because they had "self.new" methods instead of "initialize" -- which made me think it was supposed to be a class method despite creating instances.